nncf/common/composite_compression.py

```diff
@@ -216,26 +216,27 @@ def compression_stage(self) -> CompressionStage:
                 result += current_level
         return result
 
-    def load_state(self, states: List[Dict[str, object]]) -> None:
+    def get_state(self) -> Dict[str, Dict[str, Any]]:
```

Comment:
Could you please change the order of `get_state` and `load_state` method in `CompositeCompressionAlgorithmController` to unify method order across all classes?

Reply:
Done